### DISCUSSION TITLE
質問への回答へChatGPTの応答を丸投げする人がいます

### DISCUSSION THREAD

**Discussion by Verclene** [increments/qiita-discussions#443] [ヘルプ]
通報を個別にさせていただいておりますが,ほぼ表題の通りです.
ChatGPTの応答を乱投かつあまりにコンテクスト無視が目立つものがあるため,放置すると質問への回答として有用性を損なうことを危惧しています.
特に前回のような個別に対応しますというような曖昧な回答ではなく,是非運営として**対応方針**をお聞かせ願えればと思います.

**Comment by Verclene** [comment]
個人的には質問者が自らChatGPTを使えばいいところを回答者が代行するのは意味が分からんので明確に禁止した方がいいと思います.

**Reply by kskwtnk** [reply]
@Verclene 
ご意見ありがとうございます 🙏 

まず、率直な運営の感情としては「本当に辞めてほしい」と思っています
記載いただいた通り有用性を損ないますし、丸投げから生まれたコードは正しさの保証もされていません

一方で**対応方針**の提示は、この場では控えさせてください
仮にここで「ChatGPTへの丸投げは違反」と記載した場合、次のような事象が考えられます（過去の事例をもとにした予測です）

- ChatGPTを使っていない回答であっても、自分の気に入らない回答を「ChatGPTのものである」と偽って通報する
    - 通報の後、運営が強制的に対応しなかった場合に「なぜ対応しないのか、早く動くべき」といった問い合わせをする
- ガイドラインなどオフィシャルな文書で対応方針を示しても、この場に書いてある短い文章を拡大解釈して「このときはこう言っていた」と押し問答を繰り広げる

全部はNGとして、一部なら良いのか？一部ってどこからどこまでをさすのか？など、考慮すべき点が多く、やすやすと取り扱えないトピックだと考えております
ガイドラインに記載したいとは考えているのですが、今しばらくお待ちください 🙇 

> 特に前回のような個別に対応しますというような曖昧な回答

あわせてこちらのご意見へも回答します

曖昧な回答を回避することもできますが、その場合は思っているほど良い環境にはなりません
明確な閾値を設定して対応した場合、それをギリギリでクリアするスパムも、閾値を逆手に取って「自分が気に入らない人・記事を攻撃する」人も、確実に現れます
予測でしかありませんが、過去の対応履歴から、断言できます

こういった問題を吸収するためにも「個別に対応」は必要な存在です

精度が高いとは言えない記事や、いまいち文脈がずれた回答など、厳しく取り締まることは可能ですが、それはコミュニティの縮小を招きます
過去Qiitaでも技術記事以外を厳しく弾いていた時期がありましたが、良い結果は生んでいません

ご希望に沿えない回答になってしまいすみません
重ねてになりますが、今しばらくお待ちいただけますと幸いです 🙇

**Reply by Zuishin** [reply]
私は、自分の記事や、コメントした他人の記事に付きまとって「バカ」「老害」「アスペ」「キチガイ」など誰が判断しても暴言としか言えないようなことを、記事に脈絡なく数十件もしつこく書いていた人を通報し、削除してもらうのに数年かかりました。

その間の問い合わせには黙殺か機械的な返答しかなく、通報するユーザーへの対応はそれほど手間がかかるものではないという認識です。

ガイドラインに書いてあっても、その理解がおかしいんじゃないですかね。

**Reply by Verclene** [reply]
@Zuishin 
心中お察ししますがオフトピにはご配慮願います.
今回お聞きしたいのはもう少し客観かつ統一的なガイドラインは書けんものか？という一点です.
事務的・機械的な処理の速さについては議題にするつもりはありません.

先般から感じていることですが荒らしに準ずる行為に対して,コミュニティの自主努力を発揮するには**ここのサイトはあまりに弱いです**.
コミュニティの態度を明確化する意味でも,少しは運営から先手を打つような声明が出ることを期待しております.

**Reply by Zuishin** [reply]
主観的な判断で行って「ガイドラインに従いました」と言えるようあいまいな方針なんだと思うという話です。

**Reply by radian-jp** [reply]
@xrxoxcxox

> まず、率直な運営の感情としては「本当に辞めてほしい」と思っています
記載いただいた通り有用性を損ないますし、丸投げから生まれたコードは正しさの保証もされていません

であれば、discussions で以前提案した事がありますが、他の回答にも他の人がコメントを付けれるようにした方がいいんじゃないでしょうか。現状では、誰も間違った回答に対するフォローが出来ないです。また、回答に対する低評価を付ける機能はあるべきだと思います。低評価というとネガティブなイメージを持たれがちですが、回答の有用性の判断の目安にもなり、回答の質を保つためには必要な機能だと考えています。低評価の理由選択を必須にし、低評価が多い回答をデフォルトで見えなくする等すれば、運営が対応せずともChatGPTの回答を利用者側で排除する事も容易になります。

ChatGPTは正しい回答もするので、完全にガイドラインで線引きするのはおそらく難しいでしょう。逆に、正しい回答であれば排除するほどでもないかなと思います。（あくまで個人の感想です）

**Reply by yoshi389111** [reply]
> 他の回答にも他の人がコメントを付けれるようにした方がいいんじゃないでしょうか。

[Qiitaの開発ロードマップを正式に公開します！](https://github.com/increments/qiita-discussions/discussions/449)から[開発のロードマップ](https://github.com/orgs/increments/projects/8/views/1)を見るとわかると思いますが、今 In Progress になっていますよ（Qiita の中の人が頑張って作ってくれているはず）。

**Reply by radian-jp** [reply]
進行中でしたか、失礼しました。
ロードマップはチェックしていませんでした。

**Reply by kskwtnk** [reply]
@Verclene 
ガイドラインにて、AIが生成した内容についての記載を追加しますので共有します！
https://blog.qiita.com/guideline-update-2023-03/

適用は3/31からです！

**Reply by Verclene** [reply]
AIに限らず検証可能性は重要であると考えます.ご対応感謝します.
